CRITICAL - 2025-09-05 06:47:01 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "template.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('template.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('template', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(7): view('index', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 06:47:20 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "template.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('template.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('template', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(7): view('index', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-05 07:10:56 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-070829_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:10:57 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-070854_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:16:23 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-071549_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:17:13 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-070854_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:17:13 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-071549_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:20:52 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-071927_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:20:54 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-071930_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:38:19 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072931_CreateCoursesTable.php(18): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:38:19 --> mysqli_sql_exception: Invalid default value for 'joined_on' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072939_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:38:19 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072945_CreateLessonsTable.php(20): CodeIgniter\Database\Forge->createTable('lessons', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:38:19 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `q...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `q...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `q...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `q...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072954_CreateQuizzesTable.php(20): CodeIgniter\Database\Forge->createTable('quizzes', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:38:19 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-073001_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:02 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072931_CreateCoursesTable.php(18): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:02 --> mysqli_sql_exception: Invalid default value for 'joined_on' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072939_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:02 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072945_CreateLessonsTable.php(20): CodeIgniter\Database\Forge->createTable('lessons', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:02 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `q...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `q...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `q...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `q...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072954_CreateQuizzesTable.php(20): CodeIgniter\Database\Forge->createTable('quizzes', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:02 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-073001_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:38 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072931_CreateCoursesTable.php(18): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:39 --> mysqli_sql_exception: Invalid default value for 'joined_on' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072939_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:39 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072945_CreateLessonsTable.php(20): CodeIgniter\Database\Forge->createTable('lessons', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:40 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `q...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `q...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `q...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `q...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-072954_CreateQuizzesTable.php(20): CodeIgniter\Database\Forge->createTable('quizzes', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:41:40 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-073001_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:20 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075311_CreateUsersTable.php(21): CodeIgniter\Database\Forge->createTable('users', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:21 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075319_CreateCoursesTable.php(18): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:21 --> mysqli_sql_exception: Invalid default value for 'joined_on' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075357_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:21 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075403_CreateLessonsTable.php(20): CodeIgniter\Database\Forge->createTable('lessons', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:21 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `q...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `q...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `q...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `q...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075448_CreateQuizzesTable.php(20): CodeIgniter\Database\Forge->createTable('quizzes', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 07:56:21 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075455_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 08:02:08 --> mysqli_sql_exception: Table 'lms_talino.users' doesn't exist in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Seeds\UserSeeder.php(32): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 08:04:21 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075311_CreateUsersTable.php(21): CodeIgniter\Database\Forge->createTable('users', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:04:21 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075319_CreateCoursesTable.php(18): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:04:22 --> mysqli_sql_exception: Invalid default value for 'joined_on' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075357_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:04:22 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075403_CreateLessonsTable.php(20): CodeIgniter\Database\Forge->createTable('lessons', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:04:22 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `q...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `q...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `q...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `q...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075448_CreateQuizzesTable.php(20): CodeIgniter\Database\Forge->createTable('quizzes', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:04:22 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-075455_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:07:24 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080541_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 08:07:26 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080544_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 08:08:20 --> mysqli_sql_exception: Unknown table 'lms_talino.submissions' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `sub...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `sub...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `sub...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `sub...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080544_CreateSubmissionsTable.php(27): CodeIgniter\Database\Forge->dropTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:08:20 --> mysqli_sql_exception: Unknown table 'lms_talino.enrollments' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `enr...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `enr...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `enr...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `enr...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080541_CreateEnrollmentsTable.php(25): CodeIgniter\Database\Forge->dropTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->down()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:08:22 --> mysqli_sql_exception: Invalid default value for 'enrolled_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080541_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-09-05 08:08:23 --> mysqli_sql_exception: Invalid default value for 'submitted_at' in C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-PROJECTtalino\app\Database\Migrations\2025-09-05-080544_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-PROJECTtalino\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-PROJECTtalino\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-PROJECTtalino\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-09-05 08:33:15 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(10): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:34:09 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:34:35 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(10): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:34:46 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(10): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:34:47 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(10): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:35:11 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:36:10 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:36:13 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:36:15 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('home', [], true)
 4 APPPATH\Controllers\Home.php(10): view('home')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:37:09 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "home.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\Home.php(10): view('home')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:37:41 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('home', [], true)
 4 APPPATH\Controllers\Home.php(10): view('home')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:37:42 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('home', [], true)
 4 APPPATH\Controllers\Home.php(10): view('home')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-05 08:38:46 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:40:13 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(16): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:41:37 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(10): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:42:38 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(9): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:42:40 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(9): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:42:54 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(14): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:43:04 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(14): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:44:07 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: about]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('about', [], true)
 4 APPPATH\Controllers\Home.php(14): view('about')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->about()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 08:47:26 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "temple.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('temple.php')
 2 SYSTEMPATH\View\View.php(242): CodeIgniter\View\View->render('temple', [], true)
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('index', [], true)
 4 APPPATH\Controllers\Home.php(11): view('index')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
CRITICAL - 2025-09-05 09:35:10 --> Error: Class "App\Models\UserModel" not found
[Method: GET, Route: login]
in APPPATH\Controllers\Auth.php on line 26.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE311-PROJECTtalino\\public\\index.php')
